assets/queries/cloudFormation/rds_db_instance_with_deletion_protection_disabled/query.rego

```diff
@@ -0,0 +1,34 @@
+package Cx
+
+CxPolicy [ result ] {
+  document := input.document
+  resource = document[i].Resources[name]
+  resource.Type == "AWS::RDS::DBInstance"
+  properties := resource.Properties
+  properties.DeletionProtection == false  
+  
+      result := {
+                "documentId": 		input.document[i].id,
+                "searchKey":        sprintf("Resources.%s.Properties.DeletionProtection", [name]),
+                "issueType":		   "IncorrectValue",
+                "keyExpectedValue": sprintf("Resources.%s.Properties.DeletionProtection is true", [name]),
+                "keyActualValue": 	sprintf("Resources.%s.Properties.DeletionProtection is false", [name])
+              }
+}
+
+CxPolicy [ result ] {
+  document := input.document
+  resource = document[i].Resources[name]
+  resource.Type == "AWS::RDS::DBInstance"
+  properties := resource.Properties
+  exists_dp := object.get(properties, "DeletionProtection", "undefined") != "undefined"  
+  not exists_dp
+  
+      result := {
+                "documentId": 		input.document[i].id,
+                "searchKey":        sprintf("Resources.%s.Properties.DeletionProtection", [name]),
+                "issueType":		   "MissingAttribute",
+                "keyExpectedValue": sprintf("Resources.%s.Properties.DeletionProtection is defined", [name]),
+                "keyActualValue": 	sprintf("Resources.%s.Properties.DeletionProtection is undefined", [name])
+              }
+}
```

Comment:
[As a good POSIX practice](https://unix.stackexchange.com/a/18789) add end-of-file newline